calendar/exchangeapi/erAddDelegateRequest.js

```diff
@@ -13,9 +13,6 @@ Cu.import("resource://gre/modules/XPCOMUtils.jsm");
 Cu.import("resource://gre/modules/Services.jsm");
 
 Cu.import("resource://calendar/modules/calUtils.jsm");
-Cu.import("resource://calendar/modules/calAlarmUtils.jsm");
-Cu.import("resource://calendar/modules/calProviderUtils.jsm");
-Cu.import("resource://calendar/modules/calAuthUtils.jsm");
```

Comment:
@llange Curious: I see this set being removed all over. Not being familiar with TB programming myself, were they refactored into a new module? Not used at all? Didn't see anything replacing them - so just curious about the story behind the change.

Reply:
Hi @BenjamenMeyer, you're right, some of the changes are code removal.

For the specific one you're highlighting, I understood that those modules have been integrated into "calUtils.jsm" (and, by the way, are lazy-loaded only when necessary).

I'm not familiar at all with TB programming neither, all this comes from deprecation notices from the code itself that prints exactly what must be done - I only had to follow the directions ([Cf the code of calendar here](https://dxr.mozilla.org/comm-central/source/calendar/base/modules/shim/calAlarmUtils.jsm]))

In addition, a lot of the changes from one version of TB to the other is [nicely documented here](https://www.fxsitecompat.com/), detailed for each category (JS, DOM, HTML, ...) so that you can see what did change. (TB is based on the same engine as FireFox, so a lot of things are common).
The one docs that were really helpful were [this one](https://www.fxsitecompat.com/en-CA/docs/2017/expression-closure-support-has-been-removed/) and [this one](https://www.fxsitecompat.com/en-CA/docs/2017/for-each-in-loop-support-has-been-removed/).
Surely many other changes are waiting to show their impact, it's just that I didn't happen to find them.

By chance, did you have the opportunity to test the extension and see if it works for you ?

Reply:
@llange sorry, I haven't had time to test it. I don't have anything I can use for testing against as far as Exchange goes, and not sure I want to test with my "production" client. I usually look for a community blessed version as a result.